bread/contrib/document_templates/models.py

```diff
@@ -0,0 +1,22 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class DocumentTemplate(models.Model):
+    name = models.CharField(_("Name"), max_length=255)
+    model = models.ForeignKey(
+        ContentType,
+        on_delete=models.PROTECT,
+    )
+    file = models.FileField(upload_to="document_templates/")
+
+    def __str__(self):
+        return self.name
+
+
+class Document(models.Model):
+    template = models.ForeignKey(
+        DocumentTemplate, on_delete=models.CASCADE, related_name="documents"
+    )
+    file = models.FileField(upload_to="documents/")
```

Comment:
This is no longer used as well, right?
I think if we need to have a persistent file, then we also could reuse the ```ponds.models.utils.Document``` one.